### ISSUE TITLE
Get decryption key of Godot game

### ISSUE THREAD

**Issue by polybiusproxy** [vmjcv/fit-size#1]
Hey there,

This issue is unrelated to this repo, but I don't see any way to contact you.
I want to decompile a game's GDScript files, and they're encrypted, sadly.

I saw your post here (which got deleted), and I thought to contact you.
![image](https://user-images.githubusercontent.com/47796739/189734087-52aa962f-bafe-44fa-8c99-92d44895bd02.png)

I'd like to know the steps required to obtain the decryption key on IDA.
Thanks.

**Comment by Magic-KK** [comment]
Hi, did you find it?

**Comment by polybiusproxy** [comment]
> Hi, did you find it?

Yeah, found a great tool for it: https://github.com/pozm/godot-key-extract

**Comment by vmjcv** [comment]
OK.this is my idea. 
- The core c++ code is: load_byte_code. Search for this string in ida's string
- Go to the pseudo-code snippet and set a breakpoint in this code snippet.
V11 = (char *)&unk_7FF68B44FFE0;
While ( 1)

unk_7FF68B44FFE0 is the encryption key.

The direct conversion is actually similar to a 32-bit string. What I actually need is 64-bit encoding.
McQfTjWnZr4u7x! A%D*G-JaNdRgUkXp2
Just take the heinary string directly: 4D635166546A576E5A7234753778214125442A472D4A614E645267556B587032

**Comment by Magic-KK** [comment]
Thank you, I'll go back tonight and try, I can't find the load_byte_code string with IDA yesterday, I don't know why.

**Comment by Magic-KK** [comment]
Thanks, but I failed with the injection method, and the injection cannot go in.
